source/class/qx/ui/mobile/dialog/Popup.js

```diff
@@ -323,6 +320,7 @@ qx.Class.define("qx.ui.mobile.dialog.Popup", {
      * @param top {Integer} - the value the will be set to container's top style property
      */
     placeTo(left, top) {
+      this.__placed = true;
```

Comment:
`__placed` should be declared in the `members` section

Reply:
It probably should be reset someplace, but without the default of placing the popup offscreen, I'm not sure where it would be reset...

Reply:
@derrell I think one place where it should be reset is where an anchor is set.